.storybook/themes/commerce.scss

```diff
@@ -329,6 +329,61 @@
 						),
 					),
 				),
+				story-carousel: (
+					components: (
+						carousel: (
+							--slides: 1,
+						),
+						carousel-actions: (
+							display: none,
+							padding-left: var(--global-spacing-3),
+							padding-right: var(--global-spacing-4),
+							max-width: 95.8%,
+							margin-top: 0,
+						),
+						carousel-button: (
+							border-radius: var(--global-spacing-1),
+							background-color: var(--global-white),
+							box-shadow: 0px 0px 16px 0px #19191966,
+							padding: 0.75em,
+						),
+						carousel-track: (
+							gap: var(--global-spacing-5),
+							max-width: 84.7%,
+						),
```

Comment:
Just an FYI, these are now included in the default/components so you should probably not need them.

The removal of top margin is necessary?

Reply:
Reduced my own styles somewhat to just what I needed.  Yes top margin is needed to pick up the corresponding desktop breakpoint